src/components/styleElements/icons/index.js

```diff
@@ -9,6 +9,8 @@ export { Star } from './Star';
 //square icons so size is used as both width and height
 //accepts string "className" to select a className that will determine stroke color
 export { Flip } from './Flip';
+export { ChevronLeft } from './ChevronLeft';
```

Comment:
just updating the icon/index.js to reflect the changes to the ChevronLeft props and the new Filter icon